.github/workflows/dev-deploy-api.yml

```diff
@@ -41,59 +40,51 @@ jobs:
       packages: write
       deployments: write
       id-token: write
-    strategy:
-      matrix:
-        name: ['novu/api-ee', 'novu/api']
     steps:
       - uses: actions/checkout@v4
         with:
-          submodules: ${{ contains (matrix.name,'-ee') }}
+          submodules: true
```

Comment:
@SokratisVidros just to clarify, the goal with this PR is to have a separate pipeline for Community Docker Images?

Reply:
Yes.